test/utils_selection_tests.py

```diff
@@ -139,6 +139,18 @@ def get_topology_type_name(scenario_def):
         return name
 
 
+def get_topology_settings_dict(
+        monitor_class=MockMonitor, heartbeat_frequency=HEARTBEAT_FREQUENCY,
+        pool_class=MockPool, **kwargs):
+    settings = dict(
+        monitor_class=monitor_class,
+        heartbeat_frequency=heartbeat_frequency,
+        pool_class=pool_class
+    )
+    settings.update(kwargs)
+    return settings
```

Comment:
This would be more concise like:
```
 def get_topology_settings_dict(**kwargs):
    settings = dict(
        monitor_class=MockMonitor,
        heartbeat_frequency=HEARTBEAT_FREQUENCY,
        pool_class=MockPool)
    ...